kit/transport/http/middleware.go

```diff
@@ -239,3 +246,16 @@ func OrgIDFromContext(ctx context.Context) *platform.ID {
 	id := v.(platform.ID)
 	return &id
 }
+
+// reportFromCode is a helper function to determine if telemetry data should be
+// reported for this response.
+func reportFromCode(c int) bool {
+	switch {
```

Comment:
Style(?) nit: you could collapse this down to a logical expression:
```go
return (c >= 200 && c <= 299) || (c >= 500 && c <= 599)
```

Reply:
Yeah that's definitely better. Done!